src/platforms/dotnet/guides/azure-functions-worker/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Azure Functions Isolated Worker
+description: "Learn about Sentry's .NET integration with Azure Functions Isolated Worker."
+---
+
+Sentry provides an integration with Azure Functions Isolated Worker through the [Sentry.AzureFunctions.Worker NuGet package](https://www.nuget.org/packages/Sentry.AzureFunctions.Worker).
+All triggers are supported.
+
+## Install
+
+Add the Sentry dependency:
+
+```powershell {tabTitle:Package Manager}
+Install-Package Sentry.AzureFunctions.Worker -Version {{@inject packages.version('sentry.dotnet.azurefunctions.worker') }}
+```
+
+```shell {tabTitle:.NET Core CLI}
+dotnet add package Sentry.AzureFunctions.Worker -v {{@inject packages.version('sentry.dotnet.azurefunctions.worker') }}
+```
```

Comment:
The .NET CLI command should come first.

Reply:
I looked up another article and carried the order from there. Do we still want to have .NET CLI first?

Reply:
I've swapped the order. If the decision is to keep it consistent with the rest of the sample, we can revert later.